frontend/src/components/workload/Details.tsx

```diff
@@ -12,10 +12,13 @@ import {
 
 interface WorkloadDetailsProps<T extends WorkloadClass> {
   workloadKind: T;
+  name?: string;
+  namespace?: string;
```

Comment:
Do we need to support `namespace: string[]` in these?

cc @sniok

Reply:
No. This is a details view. So we just show one resource in it, which can only be in one namespace.